src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -3,45 +3,161 @@
 import java.util.List;
 
 public class MyLinkedList<T> implements MyLinkedListInterface<T> {
+    private int size;
+    private Node<T> first;
+    private Node<T> last;
```

Comment:
```suggestion
    private Node<T> head;
    private Node<T> tail;
```

Reply:
the general convention for the first node and the last one is head and tail respectively you can rename it or not, it's up to you